examples/ruby/README.md

```diff
@@ -0,0 +1,18 @@
+# Spec-Generated Ruby API Client Example
+
+This example is meant to give an example use case for OpenAPI spec generated clients.  
+It covers using some of the most popular Conjur OSS endpoints with Ruby:
+- Authenticate a user
+- Change user's own password
+- Rotate user's API key
+- Load the root policy
+- Store and retrieve a secret
+
+The `run` script is responsible for executing the example by environment-conscious means by:
+- If needed, generating a new Ruby client from the OpenAPI spec
+- If needed, spinning up a new `docker-compose` environment
```

Comment:
```suggestion
- If needed, spinning up a new `docker-compose` environment with Conjur and a Postgres database
```

Reply:
Added this to `examples/python/README.md` as it applies there as well